### DISCUSSION TITLE
How can I set up and integrate Firebase Analytics in a web project?

### DISCUSSION THREAD

**Discussion by lucas-0liveira** [capawesome-team/capacitor-firebase#450] [Q&A] [answered]
I'm working on a web project and would like to implement Firebase Analytics to track user behavior. Could someone guide me through the steps required to set up and integrate Firebase Analytics in a website or web application? What are the best practices and important considerations to keep in mind during the implementation process? Thank you in advance for your assistance!

**Comment by robingenz** [comment, marked as answer]
Thank you for your request.
You just need to add Firebase to your project and install the plugin, see https://capawesome.io/plugins/firebase/analytics/#installation.
No special configuration is required.
